blog/docker/mixed/docker-compose.yml

```diff
@@ -44,20 +44,20 @@ services:
     #deploy:
     #  replicas: 2
 
-  metrics:
-    build:
-      context: ../..
-    image: moleculer-blog
-    env_file: env
-    environment:
-      NODEID: "node-metrics"
-      SERVICES: metrics
-    links:
-      - nats
-      - redis
-    depends_on:
-      - nats
-      - redis
+  #metrics:
```

Comment:
You can remove it.

Reply:
Done in https://github.com/moleculerjs/moleculer-examples/pull/22/commits/d226aa09a9302e9dd27e55c4f6291aa5fc555a22